### DISCUSSION TITLE
Which docker image

### DISCUSSION THREAD

**Discussion by hyperknot** [onthegomap/planetiler#686] [General]
The openmaptiles repo has the following Docker command on it's README:

```
docker run -v "$(pwd)/data":/data openmaptiles/planetiler-openmaptiles:latest --force --download --area=monaco
```

meanwhile this has:

```
docker run -e JAVA_TOOL_OPTIONS="-Xmx1g" -v "$(pwd)/data":/data ghcr.io/onthegomap/planetiler:latest --download --area=monaco
```

Which one should I be using? I'd like to run a full planet conversion using the openmaptiles profile.

**Comment by acalcutt** [comment]
I'm not sure about the docker version since I always used the java release, but if you are doing full planet don't forget to take a look at https://github.com/onthegomap/planetiler/blob/main/PLANET.md . I would imagine the same options would apply to the docker release.

**Reply by hyperknot** [reply]
Thank, I'll use the java version then. Still strange that both repos ship an image and both of them are dependent on the other one.

**Reply by msbarry** [reply]
If all you want to do is generate openmaptiles then probably best just to use the planetiler-openmaptiles image. Planetiler's image is more like a Swiss Army knife that ships with a bunch of tools built-in, including the openmaptiles profile.

**Reply by hyperknot** [reply]
But is there any need for the image if all I needed to run was `apt-get install openjdk-17-jdk`? I really like docker, but in this case it's really so simple that I don't actually see the point of docker.
